webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleHandler.java

```diff
@@ -74,12 +74,14 @@ protected void wrappedService(HttpServletRequest request, HttpServletResponse re
                 sendBytes(request, response, totalPdfOutput, "application/pdf");
             } else if(ext.equals("zip")) {
                 ScrambleRequest[] scrambleRequests = ScrambleRequest.parseScrambleRequests(query, seed);
-                ByteArrayOutputStream baosZip = ScrambleRequest.requestsToZip(getServletContext(), globalTitle, generationDate, scrambleRequests, seed, generationUrl);
+                
+                System.out.println("Zip no handler");
```

Comment:
Please remove this. If you need to print something, use the logging infrastructure.

Reply:
Fixed, this was my super professional debugging.